packages/documentation/src/pages/platform/architecture/cms.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: VA.gov content management system overview
+tags: cms, drupal, headless, preview, graphql
+---
+
+# VA.gov content management system overview
+
+The VA is currently in the process of moving content from several other systems into one primary content management system, powered by Drupal. VA.gov takes a decoupled approach, where the cms is queried during the VA.gov build process and content is transformed into static html and deployed on the site. Drupal does not control any Veteran-facing rendering.
+
+In addition, there is a system that can be used to view content changes on demand, outside of the typical build process, so that content editors can preview their changes before they go live.
+
+## Existing content sources
+
+Content displayed on VA.gov exists in three primary locations:
+
+- The [vagov-content repo](https://github.com/department-of-veterans-affairs/vagov-content), in Markdown and YAML
+- Teamsite, the VA's legacy content management system
+- Drupal, the VA's current content management system
```

Comment:
The footer data, and possibly the vets-website templates might also be worth listing.